misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/picking/PickStepButton.jsx

```diff
@@ -70,6 +71,7 @@ class PickStepButton extends PureComponent {
             </div>
           </div>
         </button>
+        <PickAlternatives {...this.props} />
```

Comment:
Pls avoid spreading props on react components. It's not a good practice.

Reply:
Done